tests/lib/rules/ngInject-resolve-objects.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+const rule = require("../../../lib/rules/ngInject-resolve-objects");
+const RuleTester = require("../../../node_modules/eslint/lib/testers/rule-tester");
+const myConfig = require('./index.js');
+
+RuleTester.setDefaultConfig(myConfig);
+
+const ruleTester = new RuleTester();
+ruleTester.run("ngInject-resolve-objects", rule, {
+  valid: [
+    `Provider.bind('view', {
+        bindings: [{
+          resolve: {
+            Property: function (State) {
+              'ngInject';
+            },
+            carIds: function (State) {
+              'ngInject';
+              return State.get('car.ids');
+            }
+          }
+        }]
+     });`
+  ],
+
+  invalid: [
+    {
+      code: `Provider.bind('view', {
+                bindings: [{
+                  resolve: {
+                        BadProperty: ['State',  (State) => {
```

Comment:
Maybe make one of these tests into a regular function rather than arrow function just to test both cases.